### PR TITLE
Remove outdated kpm parts

### PR DESCRIPTION
**Pull request by chicoxyzzy**
**Summary**

According to comments and #745 kpm is an old name for yarn 

**Test plan**

N/A

### CODE REVIEW

File: src/cli/commands/login.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ async function getCredentials(config: Config, reporter: Reporter): Promise<?{
 export async function getToken(config: Config, reporter: Reporter): Promise<
   () => Promise<void>
 > {
-  const env = process.env.YARN_AUTH_TOKEN || process.env.KPM_AUTH_TOKEN || process.env.NPM_AUTH_TOKEN;
+  const env = process.env.YARN_AUTH_TOKEN || process.env.NPM_AUTH_TOKEN;
```
Comment (chicoxyzzy) [inline src/cli/commands/login.js]: not sure if this is necessary at all

**Comment by cpojer** [comment]
I'll let @bestander take care of this; it depends on the name migration at FB.

File: src/cli/commands/self-update.js [hunk 2]
```diff
@@ -35,13 +35,6 @@ export async function run(
     timeout: 5000,
   });
 
-  // while yarn is close sourced we need an auth token to be passed
-  const githubAuth0Token = process.env.YARN_AUTH_TOKEN || process.env.KPM_AUTH_TOKEN;
-  github.authenticate({
-    type: 'oauth',
-    token: githubAuth0Token,
-  });
-
```
Comment (mourner) [inline src/cli/commands/self-update.js]: The GitHub authentication part is not only a KPM part, right (there's also `YARN_AUTH_TOKEN`)? Why was it removed?

Comment (bestander) [inline src/cli/commands/self-update.js]: This authentication was used for self-update feature only.
And it was necessary only when the repo was not public yet.

Comment (bestander) [inline src/cli/commands/self-update.js]: Could you undo changes to self-update.js?
This is being fixed in https://github.com/yarnpkg/yarn/pull/676

Comment (chicoxyzzy) [inline src/cli/commands/self-update.js]: @bestander done

**Comment by bestander** [comment]
Thanks, @chicoxyzzy

**Pull request merged**